extras/http2/src/html/macros/helpers.html

```diff
@@ -0,0 +1,11 @@
+{% macro sprite(id, viewBox, classes) %}
+  {% set viewBox = viewBox or "0 0 1 1"%}
+  <span class='sprite -{{id}} {{classes}}'>
+    <svg viewBox="{{viewBox}}"><use xlink:href='images/icons.svg#{{id}}'></use></svg>
+  </span>
+{% endmacro %}
+
+
+{%- macro css(stylesheet) -%}
+	<link rel="stylesheet" href="{{ 'stylesheets/components/' ~ stylesheet ~ '/index.css' }}">
+{%- endmacro -%}
```

Comment:
extra blank line

and I know this is taken from the blog post, but a cleaner equivalent is `{% macro…` and `…endmacro %}`